client/src/containers/Gene/GeneProfile.js

```diff
@@ -118,6 +118,12 @@ class GeneProfile extends Component {
     });
   }
 
+  getDisplayName = (data) => (
+    data['gene/cgc-name'] ||
+    data['gene/sequence-name'] ||
+    data['gene/id']
```

Comment:
Should this not have a `return` statement? Or perhaps this is a feature of React/`Arrow functions` that I'm unaware of...